Assets/MixedRealityToolkit.Extensions/Socketer/Scripts/TCPConnectionManager.cs

```diff
@@ -12,6 +12,13 @@ namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.Socketer
     /// </summary>
     public class TCPConnectionManager : MonoBehaviour
     {
+        /// <summary>
+        /// Timeout interval for the socket endpoints
```

Comment:
What are the units of the timeout (miliseconds, seconds?)

Reply:
I'm going to remove this inspector behavior for now. We can make it public as needed/when having an inspector element that isn't a TimeSpan is tested.